python/lsst/ts/standardscripts/base_point_azel.py

```diff
@@ -0,0 +1,186 @@
+                        Rotator angle in mount physical coordinates (degrees).
+                    type: number
+                    default: 0.0
+                target_name:
+                    description: Name of the position.
+                    type: string
+                    default: ""
+                wait_dome:
+                    description: >-
+                        Wait for dome to be in sync with the telescope?
+                    type: boolean
+                    default: false
+                slew_timeout:
+                    description: Timeout for slew procedure (in seconds).
+                    type: number
+                    default: 240.0
+                ignore:
+                    description: >-
+                        CSCs from the group to ignore in status check. Name must
+                        match those in self.group.components, e.g.; hexapod_1.
+                    type: array
+                    items:
+                        type: string
+            required:
+                - az
+                - el
+            additionalProperties: false
+        """
+        schema_dict = yaml.safe_load(schema_yaml)
+
+        base_schema_dict = super().get_schema()
+
+        for properties in base_schema_dict["properties"]:
+            schema_dict["properties"][properties] = base_schema_dict["properties"][
+                properties
+            ]
+
+        return schema_dict
+
+    async def configure(self, config):
+        """Configure script.
+
+        Parameters
+        ----------
+        config : `types.SimpleNamespace`
+            Script configuration, as defined by `schema`.
+        """
+        self.config = config
+
+        if hasattr(self.config, "ignore"):
+            for comp in self.config.ignore:
+                if comp not in self.tcs.components_attr:
+                    self.log.warning(
+                        f"Component {comp} not in CSC Group. "
+                        f"Must be one of {self.tcs.components_attr}. Ignoring."
+                    )
+                else:
+                    self.log.debug(f"Ignoring component {comp}.")
+                    setattr(self.tcs.check, comp, False)
+
+        await super().configure(config=config)
+
+    async def assert_feasibility(self) -> None:
+        """Verify that the telescope is in a feasible state to
+        execute the script.
+        """
+
+        await self.tcs.assert_all_enabled()
+
+    async def run_block(self):
+        await self.assert_feasibility()
+
+        start_time = time.monotonic()
+        self.log.info(f"Start slew to Az/El: {self.config.az}/{self.config.el}")
+
+        await self.tcs.point_azel(
+            az=self.config.az,
+            el=self.config.el,
+            rot_tel=self.config.rot_tel,
+            target_name=self.config.target_name,
+            wait_dome=self.config.wait_dome,
+            slew_timeout=self.config.slew_timeout,
+        )
+
```

Comment:
Add a call to `self.tcs.stop_tracking()` here. By default `point_azel` will leave the telescope tracking a fixed position.